<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>365</width>
    <height>279</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Audiophiles</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget{
background:  gray;
}
#tabConnections{
background: QLinearGradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #FF9D73, stop: 1 #A63100);
}
#tabMixer{
background: QLinearGradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #FF9D73, stop: 1 #A63100);
}
#tabEqualizer{
background: QLinearGradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #FF9D73, stop: 1 #A63100);
}
#tabWifi{
background: QLinearGradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #FF9D73, stop: 1 #A63100);
}
QPushButton{
color:black;
background-color: #FF9D73;
}
QComboBox{
background: #FF9D73;
color: white;
}
QLabel{
color: white;
}
QCheckBox{
color: black;
background-color: #FF9D73;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>520</width>
     <height>16777215</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tabWifi">
       <attribute name="title">
        <string>Wifi</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="lblTxtCurrentNetwork">
          <property name="text">
           <string>Network:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lblTxtCurrentIP">
          <property name="text">
           <string>Your IP:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QPushButton" name="btnWifiRefresh">
          <property name="text">
           <string>Refresh</string>
          </property>
         </widget>
        </item>
        <item row="3" column="4">
         <widget class="QPushButton" name="btnConnectWifi">
          <property name="text">
           <string>Connect</string>
          </property>
         </widget>
        </item>
        <item row="4" column="4">
         <widget class="QPushButton" name="btnDisconnectWifi">
          <property name="text">
           <string>Disconnect</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="lblTxtAvailableNetworks">
          <property name="text">
           <string>Networks:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QComboBox" name="cmbWifi"/>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="lblNetwork">
          <property name="text">
           <string>None</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="lblIP">
          <property name="text">
           <string>0.0.0.0</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabConnections">
       <attribute name="title">
        <string>Connections</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="lblTxtReceive">
          <property name="text">
           <string>Receive:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="lblTxtSend">
          <property name="text">
           <string>Broadcast Audio:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="chkReceiveTransmit">
          <property name="text">
           <string>Receive</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QCheckBox" name="chkSendTransmit">
          <property name="text">
           <string>Transmit</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabEqualizer">
       <attribute name="title">
        <string>Equalizer</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="3">
         <widget class="QSlider" name="slideBass">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
          <property name="value">
           <number>50</number>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksAbove</enum>
          </property>
          <property name="tickInterval">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QSlider" name="slideMid">
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
          <property name="value">
           <number>50</number>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksAbove</enum>
          </property>
          <property name="tickInterval">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="QSlider" name="slideTreble">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
          <property name="value">
           <number>50</number>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksAbove</enum>
          </property>
          <property name="tickInterval">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSlider" name="slideVolEq">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
          <property name="sliderPosition">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksAbove</enum>
          </property>
          <property name="tickInterval">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QLabel" name="lblTxtTerble">
          <property name="text">
           <string>Treble</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QLabel" name="lblTxtMid">
          <property name="text">
           <string>Mid</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="lblTxtBass">
          <property name="text">
           <string>Bass</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="lblTxtVolEq">
          <property name="text">
           <string>Volume</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="lblVolEq">
          <property name="text">
           <string>100</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QLabel" name="lblBass">
          <property name="text">
           <string>50</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QLabel" name="lblMid">
          <property name="text">
           <string>50</string>
          </property>
         </widget>
        </item>
        <item row="2" column="5">
         <widget class="QLabel" name="lblTreble">
          <property name="text">
           <string>50</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>365</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
